### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by SProjects**
- Adds pre-commit git hook to enforce coding standards
- Adds Profile file and updates settings.py to enable Heroku deploy
- Adds configuration for Circle Ci and Travis CI
- Adds Travis, Coverall and Circle CI

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[flake8]
+ignore = D203
+max-line-length=119
```
Comment (amwaleh) [inline .flake8:3]: Aint this abit too long?

File: hc/settings.py [hunk 2]
```diff
@@ -109,6 +110,14 @@
             'TEST': {'CHARSET': 'UTF8'}
         }
     }
+if os.environ.get("ENVIRONMENT") == "staging":
```
Comment (amwaleh) [inline hc/settings.py]: Django used the environment variable **DEBUG** to check if we are in either Development or Production mode let use DEBUG.

File: hc/settings.py [hunk 3]
```diff
@@ -109,6 +110,14 @@
...
+    db_from_env = dj_database_url.config(conn_max_age=500)
+    DATABASES['default'].update(db_from_env)
+
+if os.environ.get("ENVIRONMENT") == "production":
```
Comment (amwaleh) [inline hc/settings.py]: This might not be necessary

**Review by amwaleh — CHANGES_REQUESTED** [review]

**Comment by amwaleh** [comment]
Rebase and Fixup the repeat commits, Then reword the commits to say what it will do 
e.g 
- [ **it will** ] `Update circle ci config`
- [ **it will** ] `Add badges for Travis, Coverall and Cicle CI`

File: .flake8 (hunk 1 above)
Comment (SProjects) [reply .flake8:3]: Discussed with TTL and left as is

**Comment by SProjects** [comment]
- Updated settings.py to use DEBUG to differentiate environments.
- Removed and reworded commits to make commit history more consistent

**Review by amwaleh — APPROVED** [review]

**Pull request merged**